RELEASE_NOTES.md

```diff
@@ -468,7 +468,11 @@ The following is a summary of the main features added to darktable
 
 ### White Balance Presets
 
-   No data for this release.
```

Comment:
This section is generated automatically, yet I have some missing data. The script gives since 4.2.1:

```
### Base Support

- Canon PowerShot SX10 IS (dng)
- Canon PowerShot SX160 IS (dng)
- Canon PowerShot SX20 IS (dng)
- Canon PowerShot SX220 HS (dng)
- Canon PowerShot SX230 HS (dng)
- Canon PowerShot SX240 HS (dng)
- Canon PowerShot SX30 IS (dng)
- Canon PowerShot SX530 HS (dng)
- Canon PowerShot SX710 HS (dng)
- Panasonic DC-GH5M2 (4:3)
- Phase One IQ180
- Phase One P45

### White Balance Presets

- Canon PowerShot SX160 IS
- Fujifilm X-H1
- OM System OM-1
- Sony DSC-RX100M7

### Noise Profiles

- Canon PowerShot SX160 IS
```

Reply:
I don't have `Sony DSC-RX100M6` & `Canon EOS R7` but I have `Canon PowerShot SX160 IS` that is missing in your list.

Reply:
Didn't we just merge the Canon EOS R7 yesterday? The Sony DSC-RX100M6 was there before indeed, just not the fine tuning ones.

Canon PowerShot SX160 IS was just a rename/cleanup, so not really worth mentioning in neither the white balance nor the noise profile section...

Reply:
> Didn't we just merge the Canon EOS R7 yesterday?

Yes, but I have checked with current HEAD.

`$ tools/check_camera_support --compare release-4.2.1 HEAD`

Reply:
Don't know why... Anyway, will drop the relnotes part of this PR...

Reply:
> > Didn't we just merge the Canon EOS R7 yesterday?
> 
> Yes, but I have checked with current HEAD.

Ah, I think this is because Canon EOS R7 is not supported via RawSpeed, and the tool cross-checks against rawspeed/data/cameras.xml? So this is a problem for all Canon CR3 cams, and both WB presets and noise profiles,,. I guess we just add these by hand for now...

Reply:
I see, this is probably the issue but I have hard tile ready the Ruby code doing that :)

Reply:
Me too 😕

Reply:
Anyway, I have added this camera in RELEASE_NOTES to ensure I wont forget it when finalizing the release in June.